Sources/AppcuesKit/Presentation/Actions/ActionRegistry.swift

```diff
@@ -72,11 +72,17 @@ internal class ActionRegistry {
         execute(transformQueue(actionInstances), completion: completion)
     }
 
-    /// Enqueue an array of experience actions instance to be executed. This version is used for post-completion actions on an experience.
+    /// Enqueue an array of experience actions instances to be executed.
```

Comment:
nit, but this one can now become private in the class, or inlined into its one usage spot on line 112